main.rb

```diff
@@ -0,0 +1,66 @@
+require './app'
+
+def main # rubocop:disable Metrics/AbcSize, Metrics/MethodLength, Metrics/CyclomaticComplexity
+  app = App.new
+
+  loop do # rubocop:disable Metrics/BlockLength
+    puts '1. List all books'
+    puts '2. List all people'
+    puts '3. Create a person'
+    puts '4. Create a book'
+    puts '5. Create a rental'
+    puts '6. List all rentals for a given person ID'
+    puts '7. Quit'
+    puts
+
+    print 'Enter your choice: '
+    choice = gets.chomp.to_i
+
+    case choice
+    when 1
+      app.list_all_books
+    when 2
+      app.list_all_people
+    when 3
+      print "Enter the person's age: "
+      age = gets.chomp.to_i
+      print "Enter the person's name (optional): "
+      name = gets.chomp
+      app.create_person(age, name: name)
+    when 4
+
+      print "Enter the book's title: "
+      title = gets.chomp
+      print "Enter the book's author: "
+      author = gets.chomp
+      app.create_book(title, author)
+    when 5
+
+      print "Enter the rental's date (YYYY-MM-DD): "
+      date = gets.chomp
+      print "Enter the book's ID: "
+      book_id = gets.chomp.to_i
+      book = app.books[book_id - 1] || (puts "Book with ID #{book_id} not found."
+                                        next)
+      print "Enter the person's ID: "
+      person_id = gets.chomp.to_i
+      person = app.people[person_id - 1] || (puts "Person with ID #{person_id} not found."
+                                             next)
+      app.create_rental(date, book, person)
+    when 6
+
+      print "Enter the person's ID: "
+      person_id = gets.chomp.to_i
+      app.list_all_rentals_for_person(person_id)
+    when 7
+      break
+    else
+      puts 'Invalid choice.'
+    end
+
+    puts
+  end
+end
```

Comment:
Your implementation of the UI is quite different from the video shared with you. I would strongly recommend all your prompts work just like the video or better!

- [ ] When I select the option to create a person I should be asked to choose if I want to create a student or teacher and based on my choice the parameters for creating the person would be shown accordingly. Kindly implement that.

-  [ ] When I select 2 to list the people created, I should have more details like name, age, class (Student or Teacher, like in the video)

- [ ] When I select the option to list the books, I should see the list books with the title and author, not just the title.

- [ ] When I select the option to create a rental I am asked to select the book ID but I have no idea what ID is being referred to because you never presented it. Kindly ensure that the create rental option works as we have in the video

  ![image](https://user-images.githubusercontent.com/54848363/216295755-656b3e95-208c-4dcd-ab9d-d8c760317785.png)

- [ ] Kindly refactor your app to work like the prompts in the video.